Add documentation about AD contextual launch feature (#9756)
opensearch-project/documentation-website: _observing-your-data/ad/index.md

    - **Confidence** estimate of the probability that the reported anomaly grade matches the expected anomaly grade. Confidence increases as the model observes more data and learns the data behavior and trends. Note that confidence is distinct from model accuracy.
    - **Last anomaly occurrence** is the time at which the last anomaly occurred.

> Underneath **Anomaly overview** or **Anomaly history** are the following sections:

Reviewer: ```suggestion
The following sections can be found under **Anomaly overview** and **Anomaly history**:
```
